routes/members-router.js

```diff
@@ -105,14 +110,15 @@ router.post('/household/invite', (req, res) => {
 });
 
 router.post('/household/accept-invite', (req, res) => {
-  const { hash } = req.body;
+  const { hash, permissionLevel } = req.body;
   const id = req.decodedToken.subject;
-  if (hash && id) {
+  if (hash && id && permissionLevel) {
     Confirmations.getById(hash).then((conf) => {
       if (conf) {
         let { member_id, household_id } = conf;
         if (member_id === id) {
-          Members.update(id, { current_household: household_id })
+          // Updates member's current household
+          Members.update(id, { current_household: household_id, permission_level: permissionLevel})
```

Comment:
By taking the permission level from the request body, this presents a security risk because with access to the backend route, someone can accept an invite and choose their own permission level. Instead we just need to extract the permission level that was stored in the confirmation table (with household_id), on line 118

Reply:
Yes, I brought this to the team's attention on Friday's stand up. Batuhan was having issues with his machine and we were just trying to get the code through. It is not breaking anything and would allow us to move forward with our objectives. However I will make the changes to get this merged asap so I can continue to make contributions in the front-end.

Reply:
I have just completed these requested changes